src/Network/Network/NetworkWatcher/ConnectionMonitor/NewAzureNetworkWatcherConnectionMonitorEndPointObjectCommand.cs

```diff
@@ -182,6 +199,20 @@ public class NewAzureNetworkWatcherConnectionMonitorEndpointObjectCommand : Conn
         [PSArgumentCompleter("Default", "Low", "BelowAverage", "Average", "AboveAverage", "Full")]
         public string CoverageLevel { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Azure Arc Network.",
+             ParameterSetName = "AzureArcNetwork")]
+        [ValidateNotNullOrEmpty]
+        public string Location { get; set; }
```

Comment:
Should we use LocationDetails or Location only ?

Help message require an update